client/stylesheets/controls.styl

```diff
@@ -151,8 +146,6 @@
       resize vertical
       table
         font-size 12px
-        background rgba(232, 255, 255, 0.75)
-        color black
```

Comment:
Can we leave the `color black`?  Bootstrap defaults to `#333`, which looks washed out to me.